plugins/woocommerce/src/Blocks/Domain/Package.php

```diff
@@ -47,12 +47,15 @@ class Package {
 	 *
 	 * @param string        $version        Version of the plugin.
 	 * @param string        $plugin_path    Path to the main plugin file.
-	 * @param FeatureGating $feature_gating Feature gating class instance.
+	 * @param FeatureGating $deprecated 	Deprecated Feature gating class.
```

Comment:
Given this is a public class, I kept the class construct signature the same, only added a default value for the `deprecated` param.